### PR TITLE
Added Home, Rules and Categories page.

### PR DESCRIPTION
**Pull request by venkateshwarans**
### Categories - https://deploy-preview-1--buzzzr.netlify.app/pages/categories.html

Site structure:
index.html
js
-- scripts.js
-- footer.js
-- header.js
css
-- global.css
-- styles.css
assets

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+body {
+  background: var(--bg-green);
+  background-image: auto;
+}
+
+.feather {
+  box-shadow: none;
+  background-color: unset;
+}
```
Comment (anik31) [inline css/styles.css:12]: You can give it a generic class name

**Review by anik31 — APPROVED** [review]
UI & code both are good.

File: css/styles.css (hunk 1 above)
Comment (venkateshwarans) [reply css/styles.css:12]: thanks for the suggestion. Actually, it is the name of my UI library. https://featherui.netlify.app/ and it is to give the feather/glass effect to a div.

**Review by nayyyhaa — APPROVED** [review]
Good coding practice is used. You can merge it.

**Pull request merged**